# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome24/gtk+/fontsel-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- gtk+-2.2.4/gtk/gtkfontsel.c.orig	2003-10-07 01:54:42.000000000 +0200
+++ gtk+-2.2.4/gtk/gtkfontsel.c	2003-10-07 01:55:07.000000000 +0200
@@ -1007,6 +1007,8 @@
   if (fontsel->face)
     {
       font_desc = pango_font_face_describe (fontsel->face);
+      pango_font_description_set_family(font_desc,
+	  pango_font_family_get_name (fontsel->family));
       pango_font_description_set_size (font_desc, fontsel->size);
     }
   else
